server/container_create.go

```diff
@@ -175,6 +175,13 @@ func (s *Server) createSandboxContainer(containerID string, containerName string
 			specgen.AddAnnotation(k, v)
 		}
 	}
+
+	// set this container's apparmor profile if it is set by sandbox
+	appArmorProfileName := GetAppArmorProfileName(sb.annotations, metadata.GetName())
+	if appArmorProfileName != "" {
+		specgen.SetProcessApparmorProfile(appArmorProfileName)
```

Comment:
It won't likely, unless there's already a profile called like that in the system I guess